drivers/eeprom/Kconfig

```diff
@@ -49,6 +49,12 @@ config EEPROM_AT25
 	help
 	  Enable support for Atmel AT25 (and compatible) SPI EEPROMs.
 
+if EEPROM_AT2X
+config EEPROM_AT2X_INIT_PRIORITY
+	int "AT2X EEPROM init priority"
+	default 90
+endif # EEPROM_AT2X
```

Comment:
Looks good, thanks!

Could we use a `depends on` line here instead? I believe that is recommended over single configs wrapped in `if ... endif`.

Reply:
Yes it is indeed

Reply:
done